### DISCUSSION TITLE
Adding a cff file for citation

### DISCUSSION THREAD

**Discussion by ChrisDahmen** [InteractiveComputerGraphics/SPlisHSPlasH#260] [Q&A] [answered]
Hello,
since this library is a good resource for prototype implementations of use case specific simulation environments it would be nice if it would have an official citation resource so that it's referenceable when using it for an implementing of a thesis or paper, such as I am doing right now for my masters thesis.
Github offers a tool to make citation easy for a repository, so a file like this would be useful, their documentation on that is [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)

As of right now I am not sure who to cite as an Author since various contributors were involved in this repository

**Comment by janbender** [comment, marked as answer]
Hi, 

That's a good point and already on my long Todo list. Here is a bibtex entry that we use in our publications: 

@Misc{SPlisHSPlasH,
  author       = {Bender, Jan and others},
  howpublished = {\ \url{https://github.com/InteractiveComputerGraphics/SPlisHSPlasH}},
  title        = {{SPlisHSPlasH Library}},
  year         = {2023},
}

Hope that helps.
